Fleece/MSVC_Compat.hh

```diff
@@ -38,6 +38,10 @@
     // WARNING: sizeof() will not work on this array since it's actually declared as a pointer.
     #define StackArray(NAME, TYPE, SIZE)    TYPE* NAME = (TYPE*)_malloca(sizeof(TYPE)*(SIZE))
 
+    #define strlcat strncat
```

Comment:
It doesn't matter what the characters are; they're just uninterpreted bytes. The problem with strncat is that, while it won't overflow the destination buffer, it won't null-terminate the string if it's too long to fit. So you end up with a corrupted string. This is the official behavior of strncat and it can't be changed, so strlcat was created as a corrected version that always null-terminates the result.